lua/weapons/e.r.c.s.lua

```diff
@@ -0,0 +1,96 @@
+SWEP.Author = "el_patito_loco"
+SWEP.PrintName = "e.r.c.s" --"Experimental Rotating Chamber Sniper"
+SWEP.Category = TASWeapons.Category
+SWEP.Author = TASWeapons.Authors.El_Patito
+SWEP.Instructions = [[Left Click shoots a powerful 12.7 mm sniper round that will destroy your enemies!!
+At the moment our engineers are working on new features which will provide more mass destruction!!
+]]
+
+SWEP.Spawnable = true
+SWEP.AdminOnly = false
+
+SWEP.ViewModel = "models/weapons/v_357.mdl"
+SWEP.WorldModel = "models/weapons/w_357.mdl"
+
+local ShootSound = Sound("weapons/mortar/mortar_fire1.wav")
+
+SWEP.FiresUnderwater = true
```

Comment:
This property doesn't seem to exist on [the `SWEP` struct](https://wiki.facepunch.com/gmod/Structures/SWEP), has this been copied from a SWEP built on a weapon framework such as TFA, and can it be removed?

Reply:
Removed fireUnderwater, it was copy paste code